src/main/java/essenmakanan/recipe/RecipeList.java

```diff
@@ -27,8 +27,8 @@ public void deleteRecipe(Recipe recipe) {
     }
 
     public void viewAllRecipes() {
-        for (Recipe recipe : recipes) {
-            System.out.println(recipe);
+        for (int i = 0; i < recipes.size(); i++) {
+            System.out.println((i + 1) + ": " + recipes.get(i).getTitle());
         }
```

Comment:
Clear and concise. Can rename function to listRecipes to standardise with listIngredients, thank you.

Reply:
I have renamed the name of this function, thanks for your advice :D